src/views/user/ChildrenCardsView/UserItem.js

```diff
@@ -90,38 +59,39 @@ function UserItem({ user, className }) {
     setState({ ...state, [event.target.name]: event.target.checked });
   };
 
-  let index = getRandomInt(1, 24);
-
+  const path = `/app/children/${childId}/log-entries`;
   return (
     <CardActionArea>
-      <Card className={clsx(classes.root, className)}>
-        <div className={classes.cardMediaWrap}>
-          <Avatar
-            alt={name}
-            src={getImgAvatar(index)}
-            sx={{
-              width: 88,
-              height: 88,
-              zIndex: 11,
-              position: 'absolute',
-              transform: 'translateY(-50%)'
-            }}
-          />
-        </div>
+      <Link to={path}>
+        <Card className={clsx(classes.root, className)}>
+          <div className={classes.cardMediaWrap}>
+            <Avatar
+              alt={name}
+              src={portraitURL}
+              sx={{
+                width: 88,
+                height: 88,
+                zIndex: 11,
+                position: 'absolute',
+                transform: 'translateY(-50%)'
+              }}
+            />
+          </div>
 
-        <CardContent className={classes.cardContent}>
-          <Typography variant="subtitle1" align="center">
-            {name}
-          </Typography>
-        </CardContent>
-        <Box display="flex" justifyContent="flex-end" width={1} padding={2}>
-          <Switch
-            checked={state.checkedA}
-            onChange={handleChange}
-            name="checkedA"
-          />
-        </Box>
-      </Card>
+          <CardContent className={classes.cardContent}>
+            <Typography variant="subtitle1" align="center">
+              {name}
+            </Typography>
+          </CardContent>
+          <Box display="flex" justifyContent="flex-end" width={1} padding={2}>
+            <Switch
+              checked={state.checkedA}
+              onChange={handleChange}
+              name="checkedA"
```

Comment:
What does 'checkedA' mean?

Reply:
Yup! That's wrong. Thanks for pointing it out.